MediaGalleryRenditions/Model/GenerateRenditions.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryRenditions\Model;
+
+use Magento\Framework\Image\AdapterFactory;
+use Magento\MediaGalleryRenditionsApi\Api\ConfigInterface;
+use Magento\MediaGalleryRenditionsApi\Api\GenerateRenditionsInterface;
+
+class GenerateRenditions implements GenerateRenditionsInterface
+{
+    /**
+     * @var AdapterFactory
+     */
+    private $_imageFactory;
```

Comment:
Please do not use the `_` property names prefix

Reply:
Okay, But may I ask why its not recommended?
Thanks

Reply:
Based on the PSR https://www.php-fig.org/psr/psr-2/#42-properties